senseapi.py

```diff
@@ -1619,6 +1620,34 @@ def StatesDefaultCheck(self):
 
 
 #================================
+#       Sharing Sensors         =
+#================================
+    def ShareSensor(self, sensor_id, commonsense_id = None, username = None):

```

Comment:
@davidsense i think it'll be nicer if we keep consistent with our way for having parameters in a call. For example, for creating a sensor, we have SensorsPost along with SensorsPost_Parameters that will generate the template of parameter for SensorsPost. The same can happen here, so instead of Only ShareSensor, we also have ShareSensorParameters method.

Another things, i'm think if we should using SensorsShare instead of ShareSensor